spaces/domains.json

```
@@ -417,5 +417,6 @@
   "vote.inuyasha.io": "yashadao.eth",
   "vote.arbibots.xyz": "arbibots.eth",
   "vote.swdao.org": "sw-dao.eth",
+  "vote.kawaii.global":"kawaii-islands.eth",
```

Please add a space here.
```suggestion
  "vote.kawaii.global": "kawaii-islands.eth",
```